src/bank.ts

```diff
@@ -48,7 +48,18 @@ export class Bank extends ServerNode {
     return await this.getPaginatedData("/blocks", options);
   }
 
-  // TODO: POST /blocks
+  /**
+   * Adds new transaction blocks to the blockchain.
+   * @param balanceLock the current balance lock
+   * @param transactions the transactions to push to the block chain
+   * @param account the account that is sending the transactions
+   */
+  async addBlocks(balanceLock: string, transactions: Transaction[], account: Account) {
+    return await this.postData(
+      "/blocks",
+      { account, balanceLock, transactions }
```

Comment:
For this, you should be setting the data in `postData` to `account.createSignedMessage` with the `balanceLock` and `transactions` as parameters to the `createSignedData` method.

Reply:
i did use initially message but then you said data so i changed it, no worries